src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.Log.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web.TokenCacheProviders.Distributed
+{
+    /// <summary>
+    /// LoggingMessage class for MsalDistributedTokenCacheAdapter.
+    /// </summary>
+    public partial class MsalDistributedTokenCacheAdapter
+    {
+        /// <summary>
+        /// LoggingMessage class for MsalDistributedTokenCacheAdapter.
+        /// </summary>
+        private static class Log
```

Comment:
>Log [](start = 29, length = 3)

Any reason this is not called logger similar to what is done for tokenacquisition? (same q in the session object)

Reply:
because token aquisition is already has a Log method for the MSAL logs.  i can change them to Logger for consistency.

Reply:
addressed [here](https://github.com/AzureAD/microsoft-identity-web/pull/1108)